django_dicom/models/patient.py

```diff
@@ -27,15 +26,6 @@ class Patient(DicomEntity):
     name_prefix = models.CharField(max_length=64, blank=True, null=True)
     name_suffix = models.CharField(max_length=64, blank=True, null=True)
 
-    if hasattr(settings, "SUBJECT_MODEL"):
-        subject = models.ForeignKey(
-            settings.SUBJECT_MODEL,
-            on_delete=models.PROTECT,
-            related_name="dicom_patient_set",
-            blank=True,
-            null=True,
-        )
-
```

Comment:
_django_dicom_ should not depend on external models, and therefore the Subject model association has been removed. This functionality is handled separately by _django_mri_.